README.md

```diff
@@ -72,16 +72,16 @@ https://openboxes.atlassian.net/wiki/spaces/OBW/pages/1719435265/Push-button+dep
 ### Install Dependencies
 
 #### Required
-* [Java 7 (must install Java 7)](https://www.azul.com/downloads/?version=java-7-lts&package=jdk)
-* [MySQL 5.7](https://downloads.mysql.com/archives/community/)
+* [Java 8 (must install Java 8)](https://www.oracle.com/pl/java/technologies/javase/javase8-archive-downloads.html) or via SDK
+* [MySQL 5.7](https://downloads.mysql.com/archives/community/) or [MariaDB 10.11.4](https://mariadb.com/kb/en/mariadb-10-11-4-release-notes/)
```

Comment:
I am not sure if it should be MySql 5.7. Shouldn't it be MySQL version 8?

Reply:
@alannadolny I don't know, locally I've always had 5.7 and it's been working even with Grails 3, so I am not sure what the suggested version should be here.
Maybe @jmiranda  @awalkowiak have suggestions?

Reply:
I think it could be all three versions 5.7, 8, and Maria, but perhaps we should start getting rid of 5.7? @jmiranda thoughts?

Reply:
See Matt's recommendations https://pihemr.atlassian.net/browse/OBS-1469

Artur is right, we support MySQL 5.7 and MySQL 8.0, as well as MariaDB 10.2 through 10.11.3. 

We do not officially support MySQL 8.1 or MariaDB 11. 

Also, there are a few issues related to the MySQL 8 upgrade that could be highlighted in the docs.
https://github.com/openboxes/openboxes/issues?q=is%3Aissue+mysql+8+is%3Aopen+

Lastly, there's at least one other PR with some changes related to the database documentation that should be reviewed.
https://github.com/openboxes/openboxes/pull/3768